autoPyTorch/pipeline/components/setup/network_backbone/ResNetBackbone.py

```diff
@@ -206,7 +206,7 @@ def __init__(
         out_features: int,
         blocks_per_group: int,
         block_index: int,
-        dropout: bool,
+        dropout: Optional[float],
```

Comment:
Can you please add a unit test where you create a pipeline with a config with dropout=0.123, and just check that the dropout of the layer is set to `0.123` to make sure this does not happen again?

Reply:
sure